modules/core/src/lib/layer-manager.js

```diff
@@ -96,7 +93,9 @@ export default class LayerManager {
         // For callback tracking and autohighlight
         index: -1,
         layerId: null
-      }
+      },
+      // Make sure context.viewport is not empty on the first layer initialization
+      viewport: viewport || new Viewport() // Current viewport, exposed to layers for project* function
```

Comment:
Keep the `DEFAULT-INITIAL-VIEWPORT` id in default fallback?

Reply:
Oh yeah, will add it back.